_posts/2025-06-30-seeds-to-system-strategy.md

```diff
@@ -0,0 +1,102 @@
+
+The team currently involved in strategic planning and development of our strategy are **Malvika Sharan, Yo Yehudi, Patricia Herterich, Irene Ramos, Riva Quiroga and Bethan Iley**.
+Alongside them, other OLS team members will be supporting the realisation of this strategy.
+
+Read the details below and reach out to them for any questions, ideas or interest in collaboration (email: [team@we-are-ols.org](mailto:team@we-are-ols.org)).
+
+## Our Mission Areas: Three Pillars
+
+Over the past few years, we have formalised our efforts into three interconnected mission areas, which we call OLS's Pillars: **Training and Mentoring, Open Research, and Open Incubator Programme**.
+Each pillar is designed to strengthen and complement the others, nurturing researchers to take their ideas to sustainable projects.
+
+### Training and Mentoring
+
+Since launching in 2020, OLS has become a recognised and reliable partner in open science training and mentoring, with a strong emphasis on involving our global community in both shaping and co-delivering our programs.
+
+* **Open Seeds**, our flagship program, has successfully run nine times, and an additional context-specific cohort of **Open Neuro Seeds**, tailored for neuroscience researchers by our Senior Resident Fellow, [Sara Villa]({% link people.md %}#saravilla). 
+* **Custom training**, like for **mentoring and Ally Skills**, or other open science training like **Nebula**, funded by NASA, which has provided two cohorts of Open Science 101 training and supported Open Seeds fellows in engaging with this initiative.
+
+These structured programs remain crucial for onboarding researchers into open science, and we have also been offering **customised training and mentorship support for various organisations**. 
+We will continue to strengthen this service area, aiming to generate revenue, providing a sustainable funding stream for Open Seeds and Nebula, which currently rely on grants and sponsorships.
+
+**What's Next?** Under "Seeds to Systems," we are developing **advanced training** for researchers and open science practitioners with projects ready to transition to the next stage.
+This program will equip them with the skills and strategies needed to design compelling project proposals, secure funding, and acquire resources to sustain their work using open-source/science solutions.
+A new **Seeds to Systems Curriculum** will be designed to directly leverage learnings and collaborations from Open Seeds.
+
+This mission will be developed in alignment with the **Open Research** and **Open Incubator** mission areas, providing critical resources and a progression path for researchers.
+More information on this will be shared later in the year.
+
+### Open Research
+
+Since 2022, OLS has been actively involved in open science research. With a purpose to ground our work in evidence, we have published reports and papers under research projects, including:
```

Comment:
```suggestion
Since 2022, OLS has been actively involved in [open science research]({% link open-research %}). With a purpose to ground our work in evidence, we have published reports and papers under research projects, including:
```